dagmc_stats.py

```diff
@@ -182,16 +180,16 @@ def get_triangle_aspect_ratio(my_core, meshset):
         
         verts = list(my_core.get_adjacencies(triangle, 0))
         
-        for side in range(3):
-            coords = my_core.get_coords(verts[side])
+        for vert in verts:
+            coords = my_core.get_coords(vert)
             coord_list.append(coords)
         
-        for length in range(3):    
-            side_lengths.append(np.linalg.norm(coord_list[length]-coord_list[indexes[length]]))
+        for side in range(3):    
+            side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
```

Comment:
definitely more readable!!  Maybe a little too clever :smile: 

This probably deserves a comment like:
```
# triangle sides go between pairs of vertices with IDs: (0,1), (1,2), (2,0)
# if interpreted as indices into a list of length 3, these can be accomplished with (0,-2), (1,-1), (2,0), or (side, 2-side)
```